src/Microsoft.Data.Analysis/PrimitiveDataFrameColumn.BinaryOperations.Combinations.ttinclude

```diff
@@ -0,0 +1,258 @@
+﻿
+<#+
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// Generated from PrimitiveDataFrameColumn.BinaryOperations.Combinations.tt. Do not modify directly
+
+public class BinaryOperationCombinations
+{
+
+    public static List<List<string>> binaryOperationCombinations = new List<List<string>>
+    {
+        new List<string> {"byte", "byte", "byte"},
```

Comment:
Instead of `List<string>`, would it be more maintainable/readable to use a custom type where you can name each field? Or even at a minimum use a `Tuple<string, string, string>`, so you know there are always 3 strings (no more, no less).

Reply:
Good idea. Made a `TypeCombination` class.